pages/dkp/kommander/2.1/backup-and-restore/index.md

```diff
@@ -0,0 +1,256 @@
+The default settings do the following:
+
+- create backups on a daily basis
+- save the data from all namespaces
+
+These default settings take effect after the cluster is created.
+If you install Konvoy with the default platform services deployed, the initial backup starts after the cluster is successfully provisioned and ready for use.
+
+#### Alternate backup schedules
+
+The Velero CLI provides an easy way to create alternate backup schedules.
+For example:
+
+```bash
+velero create schedule thrice-daily --schedule="@every 8h"
+```
+
+To change the default backup service settings:
+
+1. Check the backup schedules currently configured for the cluster by running the following command:
+
+    ```bash
+    velero get schedules
+    ```
+
+1. Delete the `velero-kubeaddons-default` schedule by running the following command:
+
+    ```bash
+    velero delete schedule velero-kubeaddons-default
+    ```
+
+1. Replace the default schedule with your custom settings by running the following command:
+
+    ```bash
+    velero create schedule velero-kubeaddons-default --schedule="@every 24h"
+    ```
+
+You can also create backup schedules for specific namespaces.
+Creating a backup for a specific namespace can be useful for clusters running multiple apps operated by multiple teams.
+For example:
+
+```bash
+velero create schedule system-critical --include-namespaces=kube-system,kube-public,kubeaddons --schedule="@every 24h"
+```
+
+The Velero command line interface provides many more options worth exploring. You can also find tutorials for [disaster recovery][velero-dr] and [cluster migration][velero-cm] on the Velero community site.
+
+### Back up on demand
+
+In some cases, you might find it necessary create a backup outside of the regularly-scheduled interval.
+For example, if you are preparing to upgrade any components or modify your cluster configuration, you should perform a backup immediately before taking that action.
+
+Create a backup by running the following command:
+
+```shell
+velero backup create BACKUP_NAME
+```
+
+### Restore a cluster
+
+Before attempting to restore the cluster state using the Velero command-line interface, you should verify the following requirements:
+
+- The backend storage, Minio, is still operational.
+- The Velero platform service in the cluster is still operational.
+- The Velero platform service must be set to a `restore-only-mode` to avoid having backups run while restoring.
+
+To list the available backup archives for your cluster, run the following command:
+
+```shell
+velero backup get
+```
+
+To set Velero to a `restore-only-mode`, modify the Velero addon in the `ClusterConfiguration` of the `cluster.yaml` file:
+
+```yaml
+addons:
+...
+- name: velero
+  enabled: true
+  values: |-
+    configuration:
+      restoreOnlyMode: true
+...
+```
+
+Then you may apply the configuration change by running:
+
+```shell
+konvoy deploy addons -y
+```
```

Comment:
This section needs to be reworked, not sure exactly how you would change the config here, it's possibly the same procedure as I showed above for setting the aws backupStorageLocation.